BraveRewardsUI/Common/DetailActionableRow.swift

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import SnapKit
+
+class DetailActionableRow: Button {
+  
+  let textLabel = UILabel().then {
+    $0.font = .systemFont(ofSize: 14.0)
+    $0.textColor = Colors.grey200
+    $0.numberOfLines = 0
+  }
+  
+  let batValueView = CurrencyContainerView(amountLabelConfig: {
+    $0.textColor = Colors.purple300
+    $0.font = .systemFont(ofSize: 14.0, weight: .semibold)
+  }, kindLabelConfig: {
+    $0.textColor = Colors.grey200
+    $0.text = "BAT"
+    $0.font = .systemFont(ofSize: 13.0)
+  })
+  
+  override var isHighlighted: Bool {
+    didSet {
+      alpha = isHighlighted ? 0.5 : 1.0
```

Comment:
Should be animated, example: https://github.com/brave/brave-rewards-ios/blob/bd17b9bfb45f670fec3b0c4a8e5cdabfaeade883/BraveRewardsUI/Common/CreateWalletButton.swift#L23-L30

Reply:
Added!